big-red-shuttle/AppDelegate.swift

```diff
@@ -12,8 +12,21 @@ import SwiftyJSON
 
 enum FileReadingError : Error { case fileNotFound }
 
-struct colorPalette{
-    static var red = UIColor(red: 0.98, green: 0.28, blue: 0.26, alpha: 1.0)
+struct Color{
+    static var red = UIColor(red:0.84, green:0.29, blue:0.21, alpha:1.0)
+    static var black = UIColor(red:0.35, green:0.35, blue:0.45, alpha:1.0)
+    static var greyedout = UIColor(red:0.56, green:0.58, blue:0.63, alpha:1.0)
+    static var lightgrey = UIColor(red:0.97, green:0.97, blue:0.97, alpha:1.0)
+
+}
```

Comment:
Let's put all these colors in a separate Extension file called UIColor+Shared.swift